sw/device/lib/testing/i2c_testutils.c

```diff
@@ -42,10 +42,16 @@ const i2c_connect_conf_t i2c_conf[] = {
     {.pins_mio_out = {kTopEarlgreyPinmuxMioOutIoa7,
                       kTopEarlgreyPinmuxMioOutIoa8},
      .pins_insel = {kTopEarlgreyPinmuxInselIoa7, kTopEarlgreyPinmuxInselIoa8},
-     .pins_peripheral_in = {kTopEarlgreyPinmuxPeripheralInI2c0Scl,
-                            kTopEarlgreyPinmuxPeripheralInI2c0Sda},
-     .pins_outsel = {kTopEarlgreyPinmuxOutselI2c0Scl,
-                     kTopEarlgreyPinmuxOutselI2c0Sda}},
+     .pins_peripheral_in =
+         {
+             kTopEarlgreyPinmuxPeripheralInI2c0Sda,
+             kTopEarlgreyPinmuxPeripheralInI2c0Scl,
+         },
+     .pins_outsel =
+         {
+             kTopEarlgreyPinmuxOutselI2c0Sda,
+             kTopEarlgreyPinmuxOutselI2c0Scl,
+         }},
```

Comment:
@a-will What are the correct pins for SDA and SCL.  Based on the hyperdebug swizzle board pinout, I believe IOA7 is SDA and IOA8 is SCL.

Reply:
You've got the right ones.

IOA8 should be SCL.
IOA7 should be SDA.

Would you mind fixing up these two files to match?
https://github.com/lowRISC/opentitan/blob/1bc8c83901c3cd8be9879d39a9d0a9b769ae32ba/hw/top_earlgrey/data/pins_cw310.xdc#L46-L47

https://github.com/lowRISC/opentitan/blob/1bc8c83901c3cd8be9879d39a9d0a9b769ae32ba/hw/top_earlgrey/dv/env/chip_if.sv#L522-L532

Reply:
Fixed.

Reply:
Thanks!